.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+persons/
```

Comment:
Why did you included the dir instead of the file?

Reply:
My mistake....I will fix it.

Reply:
Fixed!